.eslintrc.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     node: true,
   },
   rules: {
+    'jest/no-test-return-statement': 0,
```

Comment:
[Conventions] Do we really want to deactivate this rule for all our tests?

Reply:
It seems to me that yes: every async test needs that.
But we can choose to set it locally on each async test.

Reply:
I don't think so.

This test implementation won't be rejected by the initial linter rules:
```javascript
it('should call back with app', async () => {
    expect.assertions(2);

    await new Promise((resolve) => {
      const callback = (error, app) => {
        expect(error).toBeNull();
        expect(app).not.toBeNull();
        resolve();
      };
      forestExpress.init({
        ...implementation,
        opts: { callback },
      });
    });
  });
```